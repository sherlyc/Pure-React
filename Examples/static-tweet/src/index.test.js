jest.mock('react-dom', () => ({render: jest.fn()}))

import React from 'react';
import { Tweet } from './index';
import renderer from 'react-test-renderer';

it('renders', () => {
  const tree = renderer.create(
    <Tweet/>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
