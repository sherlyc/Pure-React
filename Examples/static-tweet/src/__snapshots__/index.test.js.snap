exports[`test renders 1`] = `
<div
  className="tweet">
  <img
    alt="avatar"
    className="avatar"
    src="https://www.gravatar.com/avatar/nothing" />
  <div
    className="content">
    <span
      className="name-with-handle">
      <span
        className="name">
        Your Name
      </span>
      <span
        className="handle">
        @yourhandle
      </span>
    </span>
    <span
      className="time">
      3h ago
    </span>
    <div
      className="message">
      This is less than 140 characters.
    </div>
    <div
      className="buttons">
      <i
        className="fa fa-reply reply-button" />
      <i
        className="fa fa-retweet retweet-button" />
      <i
        className="fa fa-heart like-button" />
      <i
        className="fa fa-ellipsis-h more-options-button" />
    </div>
  </div>
</div>
`;
